### ISSUE TITLE
TypeError, build_attrs() got an unexpected keyword argument 'type' in Django 1.11

### ISSUE THREAD

**Issue by prusayn** [smacker/django-filebrowser-no-grappelli#31]
When I want to create model with FileBrowseField in admin using Django 1.11 it throws me:
```
File "/home/user/.virtualenvs/project/local/lib/python2.7/site-packages/filebrowser/fields.py" in render
  42.         final_attrs = self.build_attrs(attrs, type=self.input_type, name=name)

Exception Type: TypeError at /admin/rr_website/staticfile/add/
Exception Value: build_attrs() got an unexpected keyword argument 'type'
```
I'm using newest 3.7.0 version and python 2.7.

**Comment by smacker** [comment]
I'm going to release new version this week (most probably on weekend). I'll test it with Django 1.11.
Thank you for report.

**Comment by smacker** [comment]
I released v3.7.1. It supports Django 1.11.
